OWNERS

```diff
@@ -1,43 +1,32 @@
 approvers:
   - Luap99
   - TomSweeneyRedHat
-  - alexlarsson
   - ashley-cui
   - baude
   - edsantiago
-  - flouthoc
   - giuseppe
-  - jakecorrenti
-  - jnovy
-  - jwhonce
+  - l0rd
   - lsm5
   - mheon
   - mtrmac
-  - n1hility
   - rhatdan
-  - saschagrunert
-  - umohnani8
   - vrothberg
```

Comment:
@vrothberg doesn't seem to be very active any more so should he be removed as well if we trim this down?

Reply:
Im comfy for now leaving him ... he still pops in and reviews/keeps up